[DOCS-1620] Improve accessibility of images (#1429)
wandb/docs: content/en/guides/core/registry/configure_registry.md

For example, in the proceeding image Alex inherits **Member** role privileges because they are a member of the `smle-reg-team-1` team.

> {{< img src="/images/registry/role_conflict.png" alt="Registry role conflict resolution showing how user inherits Member role through team membership" >}}

Author: ```suggestion
{{< img src="/images/registry/role_conflict.png" alt="A user inherits a Member role because they are part of a team." >}}
```
